src/models/streaming/AudioConfig.ts

```diff
@@ -0,0 +1,39 @@
+/**
```

Comment:
This comment pertains to RevAiAccount file above. Github doesn't let me leave a comment on rename only files.

Are we certain that RevAiAccount.ts should be placed into async subdirectory? It's not part of async speech.

Reply:
I added it here since it's only relevant to async responses. I can see an argument for either location